src/IsolatedTestHost/Program.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace IsolatedTestHost
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    internal static class Program
+    {
+        private static int Main(string[] args)
+        {
+            if (args.Length < 3)
```

Comment:
Check if it is != 3. Or else if you pass extra argument, it's going to be ignored without any indication.